docs/evm/near-pet-shop.md

```diff
@@ -0,0 +1,43 @@
+---
+id: near-pet-shop
+title: The NEAR Pet Shop example
+sidebar_label: Pet Shop example
+---
+
+Before you can interact with this dApp, please make sure you have a NEAR betanet account. (Note that after the NEAR mainnet launch, betanet underwent a hard fork to "start fresh” so accounts created earlier in 2020 will need to be recreated.) Please create an account following the instructions at:
+
+<a href="http://wallet.betanet.near.org" target="_blank">https://wallet.betanet.near.org</a>
+
+Then visit the NEAR Pet Shop live on betanet here:
+
+<a href="https://near-examples.github.io/near-pet-shop" target="_blank">https://near-examples.github.io/near-pet-shop</a>
+
+You'll see this on the webpage:
+
+<img loading="lazy" src="/docs/assets/evm/pet-shop-1024x591.jpg" alt="At the top of the NEAR Pet Shop there are three dogs open for adoption and one that's already adopted. The user has not logged in yet" width="1024" height="591" srcset="/docs/assets/evm/pet-shop-1024x591.jpg 1024w, /docs/assets/evm/pet-shop-300x173.jpg 300w, /docs/assets/evm/pet-shop-768x443.jpg 768w, /docs/assets/evm/pet-shop-1536x887.jpg 1536w, /docs/assets/evm/pet-shop-2048x1182.jpg 2048w" sizes="(max-width: 1024px) 100vw, 1024px">
+
+There are a total of 16 pets in a grid, associated with the 16 Ethereum addresses returned from `getAdopters`. In this screenshot, the rightmost dog named Melissa has been adopted. The button under Melissa says "Success” while the other dogs in that row say "Adopt.” Note that the buttons are disabled. This is because the user hasn't logged in yet. We won't be able to click the buttons to adopt until we're logged in and can sign transactions, but we're able to tell which dogs are adopted by instantiating `NearProvider` as "read-only” to fetch the state. (More on this in the [documentation for NEAR Web3 Provider](/docs/evm/near-web3-provider#instantiating-read-only).)
```

Comment:
Inconsistent formatting on referring to button labels: here `"Adopt"`, later on in the doc `*Adopt*`.